widgets/Esri/ArcGISRuntime/Toolkit/CoordinateConversion.cpp

```diff
@@ -0,0 +1,159 @@
+/*******************************************************************************
+ *  Copyright 2012-2020 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "CoordinateConversion.h"
+#include "ui_CoordinateConversion.h"
+
+// Toolkit headers
+#include "CoordinateEditDelegate.h"
+#include "Flash.h"
+
+// Toolkit Controller headers
+#include "Esri/ArcGISRuntime/Toolkit/CoordinateConversionController.h"
+#include "Esri/ArcGISRuntime/Toolkit/CoordinateConversionResult.h"
+#include "Esri/ArcGISRuntime/Toolkit/GenericListModel.h"
+#include "Esri/ArcGISRuntime/Toolkit/GenericTableProxyModel.h"
+
+// Qt headers
+#include <QGraphicsEllipseItem>
+#include <QItemDelegate>
+#include <QMenu>
+
+// ArcGISRuntime headers
```

Comment:
normally our include order would put the lowest level libs lower done in the order. I think it should be:

- app files if appropriate
- toolkit files
- runtime api files
- qt
- stl